### ISSUE TITLE
KerasModelImport fails while importing weights archive (Keras 2)

### ISSUE THREAD

**Issue by s78154** [deeplearning4j/deeplearning4j#5467]
Hello again,

since I updated DL4J to the new beta version, the Keras Model Import fails while importing the weights archive. I still have the same code as in issue #5051 :

```java
public void LoadFromFileSystem() {
	File modelJsonFile = new File(kerasJsonSource);
	File modelWeightFile = new File(kerasH5Source);

	MultiLayerNetwork network = null;
	try {
		network = KerasModelImport.importKerasSequentialModelAndWeights(modelJsonFile.getAbsolutePath(), modelWeightFile.getAbsolutePath(), false);
		} catch (IOException e) {
			System.err.println(e.getMessage());
			e.printStackTrace();
		} catch (InvalidKerasConfigurationException e) {
			System.err.println(e.getMessage());
			e.printStackTrace();
		} catch (UnsupportedKerasConfigurationException e) {
			System.err.println(e.getMessage());
			e.printStackTrace();
		} 
	}
}
```

Exception: 

```java
HDF5-DIAG: Error detected in HDF5 (1.10.1) thread 0:
  #000: C:\autotest\hdf5110-StdRelease-code-10vs14\build\hdfsrc\src\H5G.c line 472 in H5Gopen2(): unable to open group
    major: Symbol table
    minor: Can't open object
  #001: C:\autotest\hdf5110-StdRelease-code-10vs14\build\hdfsrc\src\H5Gint.c line 287 in H5G__open_name(): group not found
    major: Symbol table
    minor: Object not found
  #002: C:\autotest\hdf5110-StdRelease-code-10vs14\build\hdfsrc\src\H5Gloc.c line 428 in H5G_loc_find(): can't find object
    major: Symbol table
    minor: Object not found
  #003: C:\autotest\hdf5110-StdRelease-code-10vs14\build\hdfsrc\src\H5Gtraverse.c line 867 in H5G_traverse(): internal path traversal failed
    major: Symbol table
    minor: Object not found
  #004: C:\autotest\hdf5110-StdRelease-code-10vs14\build\hdfsrc\src\H5Gtraverse.c line 639 in H5G_traverse_real(): traversal operator failed
    major: Symbol table
    minor: Callback failed
  #005: C:\autotest\hdf5110-StdRelease-code-10vs14\build\hdfsrc\src\H5Gloc.c line 383 in H5G_loc_find_cb(): object 'model_weights' doesn't exist
    major: Symbol table
    minor: Object not found

Exception in thread "AWT-EventQueue-0" java.lang.RuntimeException: Unknown exception.
	at org.bytedeco.javacpp.hdf5$H5Location.openGroup(Native Method)
	at org.deeplearning4j.nn.modelimport.keras.Hdf5Archive.openGroups(Hdf5Archive.java:72)
	at org.deeplearning4j.nn.modelimport.keras.Hdf5Archive.getGroups(Hdf5Archive.java:190)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelUtils.importWeights(KerasModelUtils.java:181)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:139)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:60)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildSequential(KerasModelBuilder.java:147)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasSequentialModelAndWeights(KerasModelImport.java:253)
...
```

My `.json` and `.h5` file: [data.zip](https://github.com/deeplearning4j/deeplearning4j/files/2071059/data.zip)

After another debugging session I came to the result that the value of the variable `weightsRoot` is different in alpha and beta version. In the alpha it is `null` and in the beta it is `model_weights`. In my `.h5` file there is no node with this name. So the program throws an error.

![the h5 file](https://user-images.githubusercontent.com/34681320/40958359-5342ee26-6899-11e8-8a64-d58633fb9bc7.png)

#### Version Information

* DL4J dependencies via gradle:
deeplearning4j-core:1.0.0-beta
nd4j-native-platform:1.0.0-beta
* platform information
OS: Windows 10, 64 Bit
RAM: 8 GB
IDE: Eclipse Oxygen, Version 4.7.3a

**Comment by maxpumperla** [comment]
@s78154 thanks for reporting this, we addressed this in #5490. Essentially, what you want is:

```
network = KerasModelImport.importKerasSequentialModelAndWeights(modelPath, weightPath, false);
```

the PR fixes this. If you want to use 1.0.0-beta, you still can, using `ModelBuilder` as follows:

```
network = new KerasModel().modelBuilder()
            .modelJsonFilename((modelPath()))
            .weightsHdf5FilenameNoRoot(weightPath())
            .enforceTrainingConfig(false).buildSequential().getMultiLayerNetwork();
```

**Comment by maxpumperla** [comment]
and jfyi, the issue arises from the fact that when you store model config and weights in a single `h5` file from keras, using `model.save(...)`, you get weights stored in an h5 group called `model_weights`, which does not exist when saved separately.
